docker-compose.yml

```diff
@@ -140,3 +140,18 @@ services:
         ipv4_address: 172.29.0.50
     environment:
       - TZ=Europe/Berlin
+
+  samba:
+    image: dperson/samba
+    container_name: samba
+    networks:
+      net-demo:
+        ipv4_address: 172.29.0.32
+    command: ["-u user1;badpass", "-s public;/share", "-s user1 private;/user1;no;no;no;user1" ]
+    volumes:
+      - ./samba:/user1
+    ports:
+      - "139:139"
+      - "445:445"
```

Comment:
Die Ports zu veröffentlichen benötigen wir meiner Meinung nach nicht oder greifen wir hier vom Host oder externem namespace auf die Services zu?

Reply:
Ja, kommt in einem anderen PR. Ich brauchte die Ports zum Testen. Kann sie wieder rausnehmen.